src/psyclone/psyir/tools/definition_use_chains.py

```diff
@@ -333,7 +339,12 @@ def find_forward_accesses(self):
                     # If the control flow node is a Loop we have to check
                     # if the variable is the same symbol as the _reference.
                     if isinstance(cfn, Loop):
-                        if cfn.variable == self._reference.symbol:
+                        cfn_abs_pos = cfn.abs_position
```

Comment:
From the previous PR, but in the comment above s/could not be taken/could be 'not taken'/

Reply:
Fixed